arch/arm/configs/tsimx6_defconfig

```diff
@@ -1053,15 +1267,21 @@ CONFIG_EXFAT_FS=m
 CONFIG_NTFS_FS=m
 CONFIG_NTFS_RW=y
 CONFIG_TMPFS_POSIX_ACL=y
+CONFIG_HFS_FS=m
 CONFIG_CRAMFS=m
 CONFIG_SQUASHFS=m
+CONFIG_SQUASHFS_XATTR=y
 CONFIG_SQUASHFS_LZ4=y
 CONFIG_SQUASHFS_LZO=y
 CONFIG_SQUASHFS_XZ=y
+CONFIG_SQUASHFS_ZSTD=y
```

Comment:
SquashFS doesn't look like its enabled anywhere else. I don't think this is useful as a kitchen sink.